src/services/lansweeper/baseRequest.ts

```diff
@@ -1,30 +1,31 @@
 import { isEmpty } from "lodash";
 import { proxyFetch, adminGenericProxyFetch } from "@deskpro/app-sdk";
-import { BASE_URL, placeholders } from "../../constants";
-import { getQueryParams, getRequestBody } from "../../utils";
+import { GRAPHQL_URL, placeholders } from "../../constants";
+import { getQueryParams, getRequestBody, parseGlobalAccessToken } from "../../utils";
 import { LansweeperError } from "./LansweeperError";
 import type { Request, FetchOptions } from "../../types";
 
 const baseRequest: Request = async (client, {
   url,
   rawUrl,
```

Comment:
If `rawUrl` is passed then `url` is ignored which could be confusing to future devs. Instead, we can reduce the complexity and just have `url` and let the calling code prepend `GRAPHQL_URL` if it needs to.

Reply:
I didn't understand what you mean. We have this rawUrl, because REST API is used in the authorization, and the whole app it's GraphQL. I have not more simpler way for this. Maybe you have suggestions on how to make it easier.

Reply:
My suggestion was/is:
```
const baseRequest: Request = async (client, {
  url,
  data,
  ...
```

Something that uses rest:
```
const response = await baseRequest(client, {url: REST_BASE_URL + "foo/bar", ...});
...

Something that uses GraphQL:
```
const response = awaist baseRequest(client, {url: GRAPHQL_BASE_URL, ...});
...

Reply:
which one more easier to read and understand:
```
baseRequest(client, { url: "/me", ... });
baseRequest(client, { url: "/token", ... });
```
or
```
baseRequest(client, { url: GRAPHQL_BASE_URL + "/me", ... });
baseRequest(client, { url: REST_BASE_URL + "/token", ... });
```

Reply:
Personally, I would say the latter. It's blatantly obvious what endpoint you are calling so if I need to debug it, I know exactly where to look.

Just to be accurate, I think we're comparing the two below solutions. `rawUrl` means rest and `url` means GraphQL. Also the current code using the `const` when making a request.
```typescript
// Current
baseRequest(client, { url: "/me", ... });
baseRequest(client, { rawUrl: REST_BASE_URL + "/token", ... });
 // Suggested
baseRequest(client, { url: GRAPHQL_BASE_URL + "/me", ... });
baseRequest(client, { url: REST_BASE_URL + "/token", ... });
```